crypto/keys/hd/fundraiser_test.go

```diff
@@ -72,7 +72,6 @@ func TestFundraiserCompatibility(t *testing.T) {
 		require.Equal(t, priv[:], privB, "Expected priv keys to match")
 		var pubBFixed [33]byte
 		copy(pubBFixed[:], pubB)
-		require.Equal(t, pub, secp256k1.PubKeySecp256k1(pubBFixed), fmt.Sprintf("Expected pub keys to match for %d", i))
```

Comment:
Any alternate tests required after removing this line?

Reply:
Yeah but need to reimplement the PubKey spec according to the new library, which would take some time

Reply:
Also this hd `fundraiser_test` was removed from `cosmos-sdk` upstream